axelor-human-resource/src/main/resources/views/Timesheet.xml

```diff
@@ -189,12 +189,14 @@
 	</action-attrs>
 
 	<action-group name="action-timesheet-group-confirm">
+		<action name="action-timesheet-method-timesheet-compute-period-total"/>
```

Comment:
Better to call this method directly from the confirm method in order to avoid to have some WS for one operation.